src/tools/w3dviewer/graphicview.cpp

```diff
@@ -236,7 +236,7 @@ void CGraphicView::OnMouseMove(UINT nFlags, CPoint point)
 
         v.X = (((float)point.x - (float)(r.right >> 1)) / (float)(r.right >> 1)
                   - ((float)m_mousePos.x - (float)(r.right >> 1)) / (float)(r.right >> 1))
-            * m_radius * -1.0f;
+            * m_radius;
 
         v.Y = (((float)point.y - (float)(r.bottom >> 1)) / (float)(r.bottom >> 1)
                   - ((float)m_mousePos.y - (float)(r.bottom >> 1)) / (float)(r.bottom >> 1))

```

Comment:
On this line, the expressions are ((float)(r.bottom >> 1) - (float)point.y) and ((float)(r.bottom >> 1) - (float)m_mousePos.y)

Reply:
Up and down camera movement is incorrect in Thyme w3dview compared to stock w3dview, the above-mentioned change should fix that.

Reply:
Fixed.